website/pages/docs/advanced-topics/performance-tuning.mdx

```diff
@@ -7,6 +7,69 @@ description: Tips and tricks for improving the performance of `cloudquery sync`
+| gcp_compute_security_policies                       | project:cq-playground | 2024-07-25 13:25:45.347514 +0100 WEST | 2024-07-25 13:25:45.932167 +0100 WEST |  584.653ms |         0 |      0 |      0 |
+| gcp_compute_ssl_certificates                        | project:cq-playground | 2024-07-25 13:25:45.347976 +0100 WEST | 2024-07-25 13:25:46.045754 +0100 WEST |  697.778ms |         2 |      0 |      0 |
+| gcp_compute_ssl_policies                            | project:cq-playground | 2024-07-25 13:25:45.347981 +0100 WEST | 2024-07-25 13:25:45.858365 +0100 WEST |  510.384ms |         1 |      0 |      0 |
+| gcp_compute_subnetworks                             | project:cq-playground | 2024-07-25 13:25:45.345729 +0100 WEST | 2024-07-25 13:25:46.182778 +0100 WEST |  837.049ms |        42 |      0 |      0 |
+| gcp_compute_target_http_proxies                     | project:cq-playground | 2024-07-25 13:25:45.34819 +0100 WEST  | 2024-07-25 13:25:46.060256 +0100 WEST |  712.066ms |         2 |      0 |      0 |
+| gcp_compute_target_instances                        | project:cq-playground | 2024-07-25 13:25:45.347976 +0100 WEST | 2024-07-25 13:25:46.035427 +0100 WEST |  687.451ms |         0 |      0 |      0 |
+| gcp_compute_target_pools                            | project:cq-playground | 2024-07-25 13:25:45.348173 +0100 WEST | 2024-07-25 13:25:46.055082 +0100 WEST |  706.909ms |         0 |      0 |      0 |
+| gcp_compute_target_ssl_proxies                      | project:cq-playground | 2024-07-25 13:25:45.345808 +0100 WEST | 2024-07-25 13:25:45.774172 +0100 WEST |  428.364ms |         0 |      0 |      0 |
+| gcp_compute_target_vpn_gateways                     | project:cq-playground | 2024-07-25 13:25:45.347995 +0100 WEST | 2024-07-25 13:25:46.033584 +0100 WEST |  685.589ms |         0 |      0 |      0 |
+| gcp_compute_url_maps                                | project:cq-playground | 2024-07-25 13:25:45.345904 +0100 WEST | 2024-07-25 13:25:45.996447 +0100 WEST |  650.543ms |         3 |      0 |      0 |
+| gcp_compute_vpn_gateways                            | project:cq-playground | 2024-07-25 13:25:45.347537 +0100 WEST | 2024-07-25 13:25:46.003861 +0100 WEST |  656.324ms |         0 |      0 |      0 |
+| gcp_compute_vpn_tunnels                             | project:cq-playground | 2024-07-25 13:25:45.348019 +0100 WEST | 2024-07-25 13:25:47.007435 +0100 WEST |  1.659416s |         0 |      0 |      0 |
++-----------------------------------------------------+-----------------------+---------------------------------------+---------------------------------------+------------+-----------+--------+--------+
+```
+
+:::callout{type="info"}
+This feature is available starting from CLI version [v5.25.0](https://github.com/cloudquery/cloudquery/releases/tag/cli-v5.25.0) and plugins released on 2024-07-10 or later.
```

Comment:
Is `2024-07-10` an acceptable date format for docs? I guess people reading this are all programmers, huh. But this format is only used in Japan.